demo/mkldnn_quant/quant_aware/config.yaml

```diff
@@ -0,0 +1,22 @@
+---
+# weight quantize type, default is "channel_wise_abs_max"
+weight_quantize_type: "channel_wise_abs_max"
+# activation quantize type, default is "moving_average_abs_max"
+activation_quantize_type: "moving_average_abs_max"
+# weight quantize bit num, default is 8
+weight_bits: 8
+# activation quantize bit num, default is 8
+activation_bits: 8
+# ops of name_scope in not_quant_pattern list, will not be quantized
+not_quant_pattern: ["skip_quant"]
+# ops of type in quantize_op_types, will be quantized
+#quantize_op_types: ["conv2d", "depthwise_conv2d", "mul"]
+quantize_op_types: ["conv2d", "depthwise_conv2d", "mul", "pool2d", "elementwise_add"]
```

Comment:
The `elementwise_add` op does not support quantization yet.

Reply:
I need to check this one

Reply:
["conv2d", "depthwise_conv2d", "mul", "matmul"] is ok? pool2d does not need to quantize right? There is out_threshold now. Done